backend/package.json

```diff
@@ -20,18 +20,21 @@
   "dependencies": {
     "bcrypt": "^5.1.0",
     "cors": "^2.8.5",
+    "crypto": "^1.0.1",
```

Comment:
You didn't need to install this package. It now comes with nodeJs.

Reply:
So I should uninstall it yeah?

Reply:
Fixed